packages/custom-functions-metadata-plugin/src/customfunctionsplugin.ts

```diff
@@ -27,17 +29,14 @@ class CustomFunctionsMetadataPlugin {
 
         compiler.hooks.emit.tap(pluginName, (compilation) => {
             if (generateResult.errors.length > 0) {
-                generateResult.errors.forEach((err: string) => compilation.errors.push(inputFilePath + " " + err));
+                generateResult.errors.forEach((err: string) => compilation.errors.push(new WebpackError(inputFilePath + " " + err)));
             } else {
-                compilation.assets[this.options.output] = {
-                    source() { return generateResult.metadataJson; },
-                    size() { return generateResult.metadataJson.length; },
-                };
+                compilation.assets[this.options.output] = new sources.RawSource(generateResult.metadataJson);
             }
         });
 
         compiler.hooks.compilation.tap(pluginName, (compilation, params) => {
-            compilation.moduleTemplates.javascript.hooks.render.tap(pluginName, (source, module) => {
+            compilation.moduleTemplates.javascript.hooks.render.tap(pluginName, (source : any, module : any) => {
```

Comment:
I'm getting build errors for these changes...

Reply:
Wait, these changes seem to fix your bug!
Like, I added the ```any``` property on my own to fix this problem, so it is weird that this is happening

Reply:
After getting the latest Node and NPM, I can now build everything.